metadata.json

```diff
@@ -1,78 +1,46 @@
 [
     {
-        "name": "async-sugarcoat-04394153a7ce417b88e3fe1790a4e6a269bfebe5",
+        "name": "async-sugarcoat-04394153a7ce417b88e3fe1790a4e6a269bfebe5.js",
         "aliases": [],
         "kind": {
             "mime": "application/javascript"
         },
         "resourcePath": "async-sugarcoat-04394153a7ce417b88e3fe1790a4e6a269bfebe5.js"
     },
     {
-        "name": "async-sugarcoat-8a459c41783885dc83d30f5b7da2359091f4e607",
+        "name": "async-sugarcoat-8a459c41783885dc83d30f5b7da2359091f4e607.js",
         "aliases": [],
         "kind": {
             "mime": "application/javascript"
         },
         "resourcePath": "async-sugarcoat-8a459c41783885dc83d30f5b7da2359091f4e607.js"
     },
     {
-        "name": "brave-fix",
+        "name": "brave-fix.js",
         "aliases": [],
         "kind": {
             "mime": "application/javascript"
         },
         "resourcePath": "brave-fix.js"
     },
     {
-        "name": "brave-set-cookie-reload",
```

Comment:
Are we deliberately removing these in this PR?

Reply:
That's leftover cleanup from https://github.com/brave/adblock-resources/pull/123. I thought we had CI setup to verify these are done, but... :shrug:?